src/motor-operations.h

```diff
@@ -45,6 +46,17 @@ struct LinearSegmentSteps {
   int steps[BEAGLEG_NUM_MOTORS]; // Steps for axis. Negative for reverse.
 };
 
+// Struct used to return data about the currently executed steps
+// and the status of the auxes.
+struct PhysicalStatus {
+  // Absolute position in steps
+  int pos_steps[BEAGLEG_NUM_MOTORS];
```

Comment:
Like above, it might be interesting to clarify what 'absolute' means in this case. When we start with an un-homed machine, then home, the absolute position might be some value relative to where the machine started ?